irr/src/CIrrDeviceSDL.h

```diff
@@ -21,7 +21,6 @@
 // with enabled DirectFB, but requiring another fix at a top of SDL2.
 // We don't need DirectFB in Irrlicht/Minetest, so simply disable it here to prevent issues.
 #undef SDL_VIDEO_DRIVER_DIRECTFB
-#include <SDL_syswm.h>
```

Comment:
I think the `#undef` above was added specificially for this header so it can go too